### DISCUSSION TITLE
orin nano devkit nvme loading issue

### DISCUSSION THREAD

**Discussion by bbuce72** [OE4T/meta-tegra#1329] [Q&A]
I had a build working with "orin nano devkit nvme" but it would occasionally have the issue of "heap corrupt" described on discussion lists. I saw the update for the issue  "conf/machine: update partition layout settings for Orin Nano/NX modules
" and updated to use it. Now the NVME updates are not working with doflash or initrd.

To see where issue was,  fell back to a clean tegra-demo-distro build and was unable as well.

Using clean tegra-demo-distro:
    branch=kirkstone (commit a99fefd83cae8f4495348fe3f7db68b3a1a76fd4)
    target=jetson-orin-nano-devkit-nvme 
    image=demo-image-base 

And adding "TNSPEC_BOOTDEV:jetson-orin-nano-devkit-nvme = "nvme0n1p1"" to local.conf. 
(also tried nvme0n0p1, NVME is on  J11/PCIx4/2280/"longer" NVME slot )

Kept logs if needed but assume I missed something? 

Script of build steps attached:

[tegraDemoDistroCleanTest.txt](https://github.com/OE4T/meta-tegra/files/12361987/tegraDemoDistroCleanTest.txt)

**Comment by ichergui** [comment]
Hey @bbuce72 
I will try the latest updates from `kirkstone` and will let you know.
Which `NVME` you are using ?

**Reply by bbuce72** [reply]
One on J11 (AKA the longer 2280 slot). I drive description, its a 500GB Western Digital SN570

**Reply by ichergui** [reply]
Thanks for the quick reply.
Could you please join our gitter via the link below ? 
https://matrix.to/#/#OE4T_community:gitter.im

**Reply by bbuce72** [reply]
done
